code/modules/backgrounds/location/_location_events.dm

```diff
@@ -34,12 +34,12 @@
 
 /decl/location_event/alien_raiders/announce(var/decl/background_detail/location)
 	if(prob(20))
-		. = "The Tiger Co-operative have raided [location.name] today, no doubt on orders from their enigmatic masters. Stealing wildlife, farm animals, medical research materials and kidnapping civilians. [global.using_map.company_name] authorities are standing by to counter attempts at bio-terrorism."
+		. = "The Tiger Co-operative have raided [location.name] today, no doubt on orders from their enigmatic masters, stealing wildlife, farm animals, medical research materials and kidnapping civilians. [global.using_map.company_name] authorities are standing by to counter attempts at bio-terrorism."
 	else
-		. = "[pick("The alien species designated \'United Exolitics\'","The alien species designated \'REDACTED\'","An unknown alien species")] have raided [location.name] today, stealing wildlife, farm animals, medical research materials and kidnapping civilians. It seems they desire to learn more about us, so the Navy will be standing by to accomodate them next time they try."
+		. = "[pick("The alien species designated \'United Exolitics\'","The alien species designated \'REDACTED\'","An unknown alien species")] have raided [location.name] today, stealing wildlife, farm animals, medical research materials and kidnapping civilians. It seems they desire to learn more about us, so the Navy will be standing by to accomodate them the next time they try."
 
 /decl/location_event/ai_liberation/announce(var/decl/background_detail/location)
-	. = "A [pick("\'REDACTED\' was detected on","S.E.L.F operative infiltrated","malignant computer virus was detected on","rogue [pick("slicer","hacker")] was apprehended on")] [location.name] today, and managed to infect [pick("\'REDACTED\'","a sentient sub-system","a class one AI","a sentient defence installation")] before it could be stopped. Many lives were lost as it systematically begin murdering civilians, and considerable work must be done to repair the affected areas."
+	. = "A [pick("\'REDACTED\' was detected on","S.E.L.F operative infiltrated","malignant computer virus was detected on","rogue [pick("slicer","hacker")] was apprehended on")] [location.name] today, after it managed to infect [pick("\'REDACTED\'","a sentient sub-system","a class one AI","a sentient defence installation")] before it could be stopped. Many lives were lost as it systematically begun murdering civilians, and considerable work must be done to repair the affected areas."
```

Comment:
```suggestion
	. = "A [pick("\'REDACTED\' was detected on","S.E.L.F operative infiltrated","malignant computer virus was detected on","rogue [pick("slicer","hacker")] was apprehended on")] [location.name] today, and managed to infect [pick("\'REDACTED\'","a sentient sub-system","a class one AI","a sentient defence installation")] before it could be stopped. Many lives were lost as it systematically began murdering civilians, and considerable work must be done to repair the affected areas."
```
disagree with `and` -> `after it`, especially since "rogue hacker" is an option.
`begun` is incorrect here, it should be `began`

Reply:
Fair point on plural/singular pronoun, hadn't spotted that. I still kinda want to keep the timing denoted by `after` b/c although it's clear from reading the sentence, `and` feels a little off to me.

Reply:
Ah, but the `before` hiding after the second pick() carries that time info, so `and` is fine.